1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md

```diff
@@ -0,0 +1,17 @@
+# रिसर्च सेंसर और एक्चुएटर्स
+
+## निर्देश
+
+इस पाठ में सेंसर और एक्चुएटर्स शामिल थे। अनुसंधान और एक सेंसर और एक एक्चुएटर का वर्णन करें जिसका उपयोग IoT देव किट के साथ किया जा सकता है, जिसमें शामिल हैं:
+
+* यह क्या करता है
+* अंदर इस्तेमाल होने वाले इलेक्ट्रॉनिक्स / हार्डवेयर
+* क्या यह एनालॉग या डिजिटल है
+* इनपुट या माप की इकाइयाँ और रेंज क्या है
```

Comment:
I think a simpler word can be used for इकाइयाँ. If you think it's fine then it's ok.